### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Create Web Site_Screen Reader: Narrator/NVDA is not reading the corresponding column header while reading the description of the data inside the grid.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#133]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA

**MAS ID Violated:** MAS1.3.1

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "Create Web Site" expandable button and Press Enter.
6. Tab till "Select Folder" button and Press Enter.
7. Navigate to "Bindings" page tab.
8. Navigate to the data inside the grid and verify that screen reader is reading the corresponding column header while reading the description of the data inside the grid.

**Actual Result:**
Narrator/NVDA is not reading the corresponding column header while reading the description of the data inside the grid.

**Expected Result:**
Column header should be programmatically associated with the data inside the grid. Narrator/NVDA should read the corresponding column header while reading the description of the data inside the grid.

**Note:** This issue is observed throughout the application wherever any grid is present.

**User Impact:**
Screen Reader users will be unable to understand the data or information properly represented in the form of tabular grid.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachments:**
[Bug 133_narrator is not reading the corresponding column header while reading the description of the data inside the grid..pptx](https://github.com/Microsoft/IIS.WebManager/files/2327682/Bug.133_narrator.is.not.reading.the.corresponding.column.header.while.reading.the.description.of.the.data.inside.the.grid.pptx)
[Bug 133_NVDA is not reading the corresponding column header while reading the description of the data inside the grid..pptx](https://github.com/Microsoft/IIS.WebManager/files/2327683/Bug.133_NVDA.is.not.reading.the.corresponding.column.header.while.reading.the.description.of.the.data.inside.the.grid.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
